grants/search.js

```diff
@@ -50,6 +50,15 @@ async function buildMatchCriteria(queryParams) {
         });
     }
 
+    if (queryParams.amount) {
+        const [minAmount, maxAmount] = queryParams.amount
+            .split('|')
+            .map(parseInt);
```

Comment:
I ran into issues with `map` over `parseInt`: https://stackoverflow.com/questions/262427/why-does-parseint-yield-nan-with-arraymap – worth double checking this is okay?